glossary.rst

```diff
@@ -0,0 +1,125 @@
+  name, consisting of a common prefix, a number, and an optional suffix. For
+  example, the hosts ``cow1``, ``cow2``, and ``cow3`` could be represented by
+  the hostlist ``cow[1-3]``. 🐮
+
+.. _idset:
+
+**IDSet**
+  A compact way of representing an unordered set of integers, defined by
+  `RFC 22 <https://flux-framework.readthedocs.io/projects/flux-rfc/en/latest/spec_22.html>`_.
+
+.. _jobspec:
+
+**Jobspec**
+  A file which defines the resources, tasks, and other attributes of a single
+  program, defined by `RFC 25 <https://flux-framework.readthedocs.io/projects/flux-rfc/en/latest/spec_25.html>`_
+  and `RFC 14 <https://flux-framework.readthedocs.io/projects/flux-rfc/en/latest/spec_14.html>`_.
+  An example can be found in the `Example Jobspec
+  <https://flux-framework.readthedocs.io/projects/flux-rfc/en/latest/spec_25.html#example-jobspec>`_
+  section of RFC 25.
+
+.. _jobtap-plugin:
+
+**Jobtap Plugin**
+  A built-in or external plugin that can be loaded into the job manager broker
+  module. They are an extension of the job manager, arranging for callbacks at
+  different points in a job's life cycle. Additional information on jobtap plugins
+  can be found in :core:man7:`flux-jobtap-plugins`.
+
+.. _kvs:
+
+**KVS (Key-Value Store)**
+  A component of a Flux instance where information pertaining to the instance
+  is stored in a key-value format. It is used by Flux internally to store data
+  such as resource information, jobspec, logs, and checkpoints. Each job is
+  assigned a KVS namespace, which may also be used by users to store small
+  amounts of data.
+
+.. _limits:
+
+**Limits**
+  Maximum values that are enforced across jobs, queues, and users, such as time
+  and/or resource limits.
+
+.. _pmi:
+
+**PMI (Process Manager Interface)**
+  A standard API and wire protocol for communication between MPI runtimes and
+  resource managers.
+
+  PMI traditionally provides a key-value store with all-to-all style synchronization
+  to support the bootstrap of parallel programs such as MPI. During wire up, each
+  parallel task can bind to a network endpoint, put its network address into the PMI
+  KVS, synchronize, and then get the addresses of its network peers. All without
+  requiring the process manager (e.g. Flux) to know anything about the network
+  interconnect hardware. Brilliant!
+
+  Although there are several variations of PMI, Flux implements the original one
+  from Argonne National Lab which is documented in
+  `RFC 13 <https://flux-framework.readthedocs.io/projects/flux-rfc/en/latest/spec_13.html>`_.
+  In addition to providing PMI service to the programs it launches, Flux also uses
+  PMI for its own wire-up when it is launched by Flux or a foreign launcher.
+
+  PMI was an unfortunate three letter acronym choice, given that MPI was already a
+  thing. This is known - no whining.
+
+.. _policy:
+
+**Policy**
+  Attributes of a Fux queue that define behavior of the queue, such as time or
+  resource limits.
```

Comment:
Fux -> Flux

Reply:
Thanks, just force-pushed a change to fix this typo.